includes/Data/Patterns.php

```diff
@@ -218,4 +226,26 @@ public static function set_theme_step_patterns( $step, $slug ) {
 				);
 		}
 	}
+
+	public static function get_count_of_patterns() {
+
+		 $active_theme = ( \wp_get_theme() )->get( 'TextDomain' );
+		 $theme_steps  = self::get_theme_step_patterns()[ $active_theme ];
```

Comment:
```suggestion
		 $theme_step_patterns  = self::get_theme_step_patterns();
		 $active_theme_patterns = isset($theme_step_patterns[ $active_theme ]) ?  isset($theme_step_patterns[ $active_theme ]) : array();
```
To prevent PHP errors when someone tries to access onboarding with an active theme we do not support.

Reply:
Sure